File: MSA-Phase-2/MSA-Phase-2/Models/Book.cs
namespace MSA_Phase_2.Models
{
    public class Book
    {
        public int BookId { get; set; }
        public int AuthorId { get; set; }
        public string BookName { get; set; }
        public string CoverImageS { get; set; }
        public string CoverImageM { get; set; }
        public string CoverImageL { get; set; }
        public int Year { get; set; }
        // Book has a single author
        public Author Author { get; set; }
        // Book has multuiple reviews
        public ICollection<Review> Reviews { get; set; }

    }
}

File: frontend/src/models/Book.ts
import { Author } from './Author';
import { Review } from './Review';


export interface Book {
    bookId: number;
    authorId: number;
    bookName: string;
    coverImageS: string;
    coverImageM: string;
    coverImageL: string;
    year: number;
    author: Author;
    reviews: Review[];
}